test/functional/fixtures/reporter/testcafe-fixtures/index-test.js

```diff
@@ -25,6 +25,7 @@ async function errorCheck (t) {
 
 test('Simple test', async t => {
     await t.wait(1);
+    await t.report();
```

Comment:
This call should be moved to the `Run t.report action with object val` test.

Reply:
This change was made for another test: https://github.com/DevExpress/testcafe/blob/master/test/functional/fixtures/reporter/test.js#L1166
We need to run this action in order to raise reportData event.